tslint.json

```diff
@@ -21,7 +21,8 @@
       -1,
       0,
       1,
-      2
+      2,
+      3
```

Comment:
are we going to go back and remove all the `tslint:disable-next-line: no-magic-number`? 
Or just wait for the tslint to eslint formal migration?

Reply:
I think it might be best to just wait for the migration.